docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
+
+The Faker connector generates realistically looking random data matching a
+specific schema.
```

Comment:
schema is kinda a bad term in this context .. since its not the catalog/schema/table you are referring to right?

Maybe we can say something like "matching a defined structure"?